analytics/views.py

```diff
@@ -1257,9 +1257,18 @@ def get_confirmations(
     return confirmation_dicts
 
 
+def get_realm_admin_emails_as_string(realm: Realm) -> str:
+    return ", ".join(
+        realm.get_human_admin_users()
+        .order_by("delivery_email")
+        .values_list("delivery_email", flat=True)
+    )
+
+
 def set_additional_realm_info(realm: Realm) -> None:
-    realm.customer = get_customer_by_realm(realm)
+    realm.admin_emails_string = get_realm_admin_emails_as_string(realm)
```

Comment:
I think the change to a string is good, but the above comment on not adding attributes to `Realm` objects applies here too -- we should be building a `context` object, not a `realm` object.